### ISSUE TITLE
How app homepages in main buckets are checked if they are malicious

### ISSUE THREAD

**Issue by quantuumsnot** [lukesampson/scoop#3618]
Asking just for the clarity because 5 mins ago checked most of the games from [scoop-games](https://github.com/Calinou/scoop-games/tree/master/bucket) bucket and some were marked and reported for spreading malware. For example http://fte.triptohell.info

**Comment by SonOfDiablo** [comment]
It seems like the app homepages (and other links in the manifest) isn't checked, Scoop only check the hashes that is provided in the manifest. (as far as I can tell at least, please if I'm wrong, someone tell me).

I wrote up a little python script (borrowed code from https://github.com/TheresAFewConors/Sooty) and grabbed the list of apps in the scoop-games bucket.

Here is what I found: https://pastebin.com/Y7qXaQMD

I grabbed results from the homepage provided in the manifests and issued a rescan if the scan was older than 7 days or if there was no scan.
I also looked up the file hashes now that I was at it (I haven't yet figured out a feasible way to submit files for scan yet)

I hope this helped you at least a litte.

**Comment by r15ch13** [comment]
Homepages are not checked. Files also don't get checked. Scoop just queries virustotal if you run `scoop virustotal <app>`.

@SonOfDiablo there are some [scripts available](https://support.virustotal.com/hc/en-us/articles/115002146469-API-scripts). But you can also simply submit the download URL. See: https://developers.virustotal.com/reference#url-scan

**Comment by SonOfDiablo** [comment]
@r15ch13 yea I do submit urls for scan, I just haven't figured out how to submit files since there is a 32mb limit unless you request a special upload url that can then handle up to 200mb.

Also I'm not sure what you mean with submit them? I thought they only queried VirusTotal with the hashes?
If I run an app that isn't on VirusTotal with `scoop virustotal <app>` I get something like this:
`WARN  dwarf-fortress-lnp: not found: manually submit http://dffd.bay12games.com/download.php?id=7622&f=PeridexisErrant%27s+Starter+Pack+0.44.12-r07.zip#/dl.7z`
Telling me to manually download and upload the file to VirusTotal.
(Maybe I'm just misunderstanding what you are saying)

**Comment by r15ch13** [comment]
Yeah, you're right. It will not submit the file, just the URL. Thought VirusTotal would download it.
